src/RadioButton.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { color } from 'styled-system'
+import Icon from './Icon'
+
+const RadioButton = ({
+  className,
+  theme,
+  onClick,
+  isSelected,
+  disabled,
+  radioButtonText
+}) => {
+  const radioIconName = isSelected ? 'radioFilled' : 'radio'
+  const radioIconColor = isSelected ? 'blue' : 'borderGray'
+
+  function handleClick() {
+    if (disabled || isSelected) return
+    onClick()
+  }
+
+  return (
+    <RadioButtonWrap
+      className={className}
+      onClick={handleClick}
+      isSelected={isSelected}
+      disabled={disabled}
+      theme={theme}
+    >
+      <RadioIconWrap>
```

Comment:
This could possibly be a use case for `<IconButton />`.  Take a look at the documentation and see if you think it could make this a little simpler.  It would at least include a pointer cursor, and might require less wrapping than you have currently

Reply:
If `IconButton` takes `children` so we can pass in the `Text` in that would be good. 
But right now,  I have to have `label` wrapper to wrap around `Icon` and `Text` and all the hover, cursor logics are done in the wrapper.